modin/pandas/test/test_dataframe.py

```diff
@@ -2968,190 +2968,66 @@ def test_hasattr_sparse(self, data):
-                    names=axis_names,
-                )
-                modin_df_multi_level.index = new_idx
-                pandas_df_multi_level.index = new_idx
-            else:
-                new_col = pandas.MultiIndex.from_tuples(
-                    [(i // 4, i // 2, i) for i in range(len(modin_df.columns))],
-                    names=axis_names,
-                )
-                modin_df_multi_level.columns = new_col
-                pandas_df_multi_level.columns = new_col
-
-            for level in list(range(levels)) + (axis_names if axis_names else []):
-                try:
-                    pandas_multi_level_result = pandas_df_multi_level.all(
-                        axis=axis, bool_only=bool_only, level=level, skipna=skipna
-                    )
-
-                except Exception as e:
-                    with pytest.raises(type(e)):
-                        modin_df_multi_level.all(
-                            axis=axis, bool_only=bool_only, level=level, skipna=skipna
-                        )
-                else:
-                    modin_multi_level_result = modin_df_multi_level.all(
-                        axis=axis, bool_only=bool_only, level=level, skipna=skipna
-                    )
-
-                    df_equals(modin_multi_level_result, pandas_multi_level_result)
-
-    @pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-    @pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
     @pytest.mark.parametrize(
         "skipna", bool_arg_values, ids=arg_keys("skipna", bool_arg_keys)
     )
     @pytest.mark.parametrize(
         "bool_only", bool_arg_values, ids=arg_keys("bool_only", bool_arg_keys)
     )
-    def test_any(self, data, axis, skipna, bool_only):
-        modin_df = pd.DataFrame(data)
-        pandas_df = pandas.DataFrame(data)
-
-        try:
-            pandas_result = pandas_df.any(axis=axis, skipna=skipna, bool_only=bool_only)
-        except Exception as e:
-            with pytest.raises(type(e)):
-                modin_df.any(axis=axis, skipna=skipna, bool_only=bool_only)
-        else:
-            modin_result = modin_df.any(axis=axis, skipna=skipna, bool_only=bool_only)
-            df_equals(modin_result, pandas_result)
-
-        try:
-            pandas_result = pandas_df.any(axis=None, skipna=skipna, bool_only=bool_only)
-        except Exception as e:
-            with pytest.raises(type(e)):
-                modin_df.any(axis=None, skipna=skipna, bool_only=bool_only)
-        else:
-            modin_result = modin_df.any(axis=None, skipna=skipna, bool_only=bool_only)
-            df_equals(modin_result, pandas_result)
+    def test_all_any(self, skipna, bool_only):
+        data = {
+            "int_col": [1, 2, np.NaN, 4],
+            "float_col": [np.NaN, 9.4, 10.1, np.NaN],
+            "str_col": ["a", np.NaN, "c", "d"],
+            "bool_col": [False, True, np.NaN, np.NaN],
+        }
```

Comment:
Many errors may be hidden when we use small dimensioned data. In that case, the whole frame will be located in a single partition, so we're not testing parallel implementation at all. 

Can we use here more dimensional data? I think something about `256x64` will be enough to locate frame in a few partitions

Reply:
Is there a way to force set the number of partitions?

Reply:
it's already explicitly setted at the beginning of the tests
https://github.com/modin-project/modin/blob/843816ecbccb6435ba7f29688301f4198277cd27/modin/pandas/test/test_dataframe.py#L67

Reply:
I'm talking about being able to get multiple partitions on a small dataset.

Reply:
No.
For default minimum chunk size is `32x32` (and that default value partition manager uses to [compute chunk size based on `DEFAULT_NPARTITIONS`](https://github.com/modin-project/modin/blob/843816ecbccb6435ba7f29688301f4198277cd27/modin/engines/base/frame/partition_manager.py#L373-L399)) and we have no ability to change it
https://github.com/modin-project/modin/blob/843816ecbccb6435ba7f29688301f4198277cd27/modin/data_management/utils.py#L33-L41